try-it-out/docs/pages/deploy-devops-tools.rst

```diff
@@ -0,0 +1,60 @@
+.. _Deploy Devops Tools:
+-----------------------
+Deploy The Devops Tools
+-----------------------
+
+In this section, you will be deploying all of the DevOps tools that we will be using within this tutorial. This consists of deploying a Jenkins instance, a SonarQube server, and a local (insecure) Docker registry. 
+
+.. note:: The tools that we will be utilizing throughout this tutorial are not fully secure and thus not production-ready. They are meant to just be a showcase of what SDP itself can do. 
+
+To begin, clone our SDP-Labs_ GitHub repository, which includes all of the files you'll need to use throughout our guide. 
+
+.. _SDP-Labs: https://github.com/boozallen/sdp-labs
+
+.. code-block:: bash
+
+   ## cloning via ssh
+   git clone git@github.com:boozallen/sdp-labs.git
+   ## cloning via https
+   git clone https://github.com/boozallen/sdp-labs.git
+
+**If you haven't already**, use one of the above commands to clone the repository to your computer. 
+
+In your terminal, navigate to the inside of the ``sdp-labs`` directory that you just cloned from GitHub.
+
+Run the following command in your terminal from the top-level directory of the ``sdp-labs`` directory to deploy the Docker containers using the `provided docker-compose.yaml file`_ within the ``try-it-out`` directory. For some more information about docker-compose files, visit `this page`_.
+
+.. _provided docker-compose.yaml file: https://github.com/boozallen/sdp-labs/blob/master/try-it-out/docker-compose.yaml
+
+.. code-block:: bash
+
+   docker-compose -f ./try-it-out/docker-compose.yaml up -d
```

Comment:
does docker-compose up build images by default if they're not found?

Reply:
Yes! It generally gives the following error: 

"WARNING: Image for service jenkins was built because it did not already exist. To rebuild this image you must use `docker-compose build` or `docker-compose up --build`."

If you make a change to how Jenkins should be built (i.e. the configure.groovy file), you will need to use docker-compose up --build to capture those changes in a new Docker image.

Reply:
I've modified the command to be:

docker-compose -f ./try-it-out/docker-compose.yaml up --build -d --scale sonar-scanner=0